packages/form/src/SelectRow.js

```diff
@@ -59,68 +60,92 @@ function getValueToLabelAvatarMap(fromChildren = []) {
 class SelectRow extends PureComponent {
     static propTypes = {
         label: PropTypes.node.isRequired,
-        asideAll: PropTypes.string,
-        asideNone: PropTypes.string,
+        asideAllLabel: PropTypes.oneOfType([
+            PropTypes.string,
+            PropTypes.bool, // can pass false to disable 'All' label
+        ]),
+        asideNoneLabel: PropTypes.string,
         asideSeparator: PropTypes.string,
         disabled: PropTypes.bool,
         // <SelectList> props
         multiple: SelectList.propTypes.multiple,
-        values: SelectList.propTypes.values,
-        defaultValues: SelectList.propTypes.defaultValues,
+        value: SelectList.propTypes.value,
+        defaultValue: SelectList.propTypes.defaultValue,
         onChange: PropTypes.func,
         // from formRow()
         ineditable: PropTypes.bool,
         rowProps: rowPropTypes,
     };
 
     static defaultProps = {
-        asideAll: 'All',
-        asideNone: '(Unset)',
+        asideAllLabel: 'All',
+        asideNoneLabel: '(Unset)',
         asideSeparator: ', ',
         disabled: false,
         // <SelectList> props
         multiple: SelectList.defaultProps.multiple,
-        values: SelectList.defaultProps.values,
-        defaultValues: SelectList.defaultProps.defaultValues,
+        value: SelectList.defaultProps.value,
+        defaultValue: SelectList.defaultProps.defaultValue,
         onChange: () => {},
         // from formRow()
         ineditable: false,
         rowProps: {},
     };
 
     state = {
-        popoverOpen: false,
+        isPopoverOpen: false,
         valueLabelMap: getValueToLabelAvatarMap(this.props.children),
-        cachedValues: this.props.values || this.props.defaultValues,
+        cachedValue: this.getInitialValue(),
     };
 
     componentWillReceiveProps(nextProps) {
         this.setState({
             valueLabelMap: getValueToLabelAvatarMap(nextProps.children),
         });
 
+        warning(
+            this.getIsControlled(this.props) === this.getIsControlled(nextProps),
```

Comment:
```suggestion
            this.getIsControlled(this.props) !== this.getIsControlled(nextProps),
```

Reply:
The first parameter of `warning()` should be "correct case",  it will popup warning if this value is falsy.

ref: https://github.com/BerkeleyTrue/warning#usage